tests/cypress/support/index.js

```diff
@@ -39,6 +39,17 @@ Cypress.Cookies.defaults({
 });
 
 before(() => {
+  // This is needed for search to deploy RedisGraph upstream. Without this search won't be operational.
+  cy.exec('oc get pod -n open-cluster-management | grep search-redisgraph-0 | grep Running', {failOnNonZeroExit: false}).then(result => {
+      if (result.code == 0){
+        cy.task('log', 'Redisgraph pod is running.')
```

Comment:
To enable the `log` task we need to add this to `/cypress/plugins/index.js`

```js
  on('task', {
    log(message) {
      console.log(message)

      return null
    },
  })
```

Reply:
Is there a reason not to use `cy.log`?

Reply:
`cy.log()` doesn't print to the terminal so you can't see it when running headless in Travis.